packages/zoe/src/zoeService/zoeStorageManager.js

```diff
@@ -120,6 +120,7 @@ export const makeZoeStorageManager = (
     uncleanIssuerKeywordRecord,
     instance,
     contractBundleCap,
+    instanceBaggage,
```

Comment:
When the baggage parameter is not optional, all call sites need to change anyway, so better to make the baggage parameter be first.

Reply:
done